modules/nw-ovn-kubernetes-migration-about-live.adoc

```diff
@@ -0,0 +1,166 @@
+** The Machine Config Operator cordons, drains, and reboots the node.
+** When the node is back up, the `br-ex` bridge is created, which indicates that the node is ready for OVN-Kubernetes. As a result, the local zone OVN-Kubernetes can be started. The `ovnkube-node` container wrapper script also adds the node annotation `k8s.ovn.org/node-subnets` according to the HostSubnet CR of this node. 
++
+On the local node, OVN-Kubernetes adds all OpenShift SDN nodes as hybrid overlay nodes to the local zone, and all OVN nodes as OVN-IC remote zones. On a remote OVN node, OVN-Kubernetes removes a hybrid overlay node and adds an OVN-IC remote zone. Pods are recreated on the node using OVN-Kubernetes as the default CNI plugin. 
++
+This process is repeated for each node until all nodes have been applied to the new MachineConfig and converted to OVN-Kubernetes. 
+
+. The Cluster Network Operator watches the machine config pools and waits until the new MachineConfig is applied to all nodes. It then sets the `NetworkTypeMigrationTargetCNIInUse` condition to `True` in the status of the `network.config` CR.
+
+. The Cluster Network Operator removes the `spec.migration` field of the `network.operator` custom resource. It then triggers the CNO to perform the following actions:
+** Delete the `openshift-sdn` DaemonSets and the related resources, such as custom resources, ConfigMaps, and so on. 
+** Redeploy OVN-Kubernetes in `normal` mode, so that the migration mode script is not present, hybrid overlay is disabled, host subnet allocation is enabled, and so on. 
+** Remove the hybrid overlay related labels and node hybrid overlay annotations from the nodes.
+
+. The Cluster Network Operator waits until OVN-Kubernetes is redeployed and sets the `NetworkTypeMigrationOriginalCNIPurged` condition to `True` in the status of the `network.config` CR. Then, it sets the `NetworkTypeMigrationInProgress` condition to `False` and the reason as `NetworkTypeMigrationCompleted` in the status of the `network.config` CR. This indicates that the migration has successfully completed. 
+
+== Prerequisites for live migration
+
+The following prerequisites must be met before migrating to the OVN-Kubernetes network plugin from the OpenShift SDN plugin:
+
+* A cluster has been configured with the OpenShift SDN CNI network plugin in the network policy isolation mode.
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+* You have created a recent backup of the etcd database is available.
+
+* The cluster is in a known good state without any errors.
+
+* On all cloud platforms after updating software, a security group rule must be in place to allow UDP packets on port `6081` for all nodes.
+
+* Cluster administrators must disable the following custom resources:
++
+** Egress Router
+** Egress IP
+** Egress Firewalls
+** Multicast
```

Comment:
Only the egress router configuration shall be removed. The configuration for other features can be migrated automatically after the live migration is done. The other features just don't work during the live migration.

Reply:
I'm not sure if I'm understanding 100% correctly: 

Do egress IP, egress firewalls, and multicast need to be removed by the administrator before beginning the live procedure? or are they removed automatically?

is the egress router removed automatically? or does it need to be removed by the administrator?

Thanks!

Reply:
The admin shall remove the egress router configuration. The configuration for egress IP, egress firewalls, and multicast can be migrated automatically after the live migration is done. So users don't need to remove them.

Reply:
Multicast, egress IP addresses, and egress firewalls are not disabled. They just cannot work as expected.